### ISSUE TITLE
Julia 1.10.1+ breaks automatic installation of AutoEIS

### ISSUE THREAD

**Issue by benjamin5988** [AUTODIAL/AutoEIS#95]
What is your workaround for this? I cannot install or run v0.0.25 for this reason. whenever i try to import autoeis it segfaults

**Comment by ma-sadeghi** [comment]
Good catch. We're aware of the issue, but didn't get the chance to fix it yet. For now, find the `autoeis` directory, edit `julia_helpers.py` and replace:

```python
def install_julia():
    """Installs Julia using juliapkg."""
    # Importing juliacall automatically installs Julia using juliapkg
    import juliacall
```

with:

```python
def install_julia():
    """Installs Julia using juliapkg."""
    # Importing juliacall automatically installs Julia using juliapkg
    import juliapkg
    juliapkg.require_julia("=1.10.0")
    juliapkg.resolve()
    import juliacall
```

PS. If you don't know how to locate `autoeis` directory, just Google "where are Python packages stored + your_operating_system".

**Comment by ma-sadeghi** [comment]
`juliacall` has been patched to prevent installing Julia > 1.10.0, so this can be closed.
